### PR TITLE
marabou-v1.0 typos and warnings

### PR DESCRIPTION
**Pull request by youssefaz94**

### CODE REVIEW

File: frontend/src/components/MasterHead.js [hunk 1]
```diff
@@ -9,7 +8,7 @@ class MasterHead extends React.Component {
         <div className="container h-100">
             <div className="row h-100 align-items-center justify-content-center text-center">
                 <div className="col-lg-10 align-self-end">
-                    <h1 className="text-uppercase text-white font-weight-bold">We help you brige the AI gap for your business</h1>
+                    <h1 className="text-uppercase text-white font-weight-bold">We help you bridge the AI gap for your business</h1>
```
Comment (youssefaz94) [inline frontend/src/components/MasterHead.js:11]: @mmarouen typo

**Comment by youssefaz94** [comment]
@mmarouen I have improved the css loading, so there is no intermediate phase where the css isn't already loaded

**Comment by youssefaz94** [comment]
fixed the display of the clothing Classifier view

**Comment by mmarouen** [comment]
ok

**Pull request merged**